test.js

```diff
@@ -1,46 +1,27 @@
 import { equal } from 'assert';
 import mdast from 'mdast';
 import mdastTextr from './index';
-import base from 'typographic-base';
 
-const t = (text, mdastOptions) =>
-  mdast()
-    .use(mdastTextr, mdastOptions)
-    .process(text)
-    .trim();
+// textr plugin — just function to replace triple dots to ellipses
+const ellipses = input => input.replace(/\.{3}/gim, '…');
 
-describe('mdast-textr', () => {
+const text = `
+## spread operator...
 
-  it('should return original text', () =>
```

Comment:
what if smth happend and mdast-textr will transform text? for example, somebody attached any transformer as default.